packages/ods/src/components/skeleton/documentation/migration.from.17.x.mdx

```diff
@@ -0,0 +1,27 @@
+## Attributes changes
```

Comment:
This doc is a really important part of the transition adoption to v18, treat each attribute separately and add a dedicated example for each